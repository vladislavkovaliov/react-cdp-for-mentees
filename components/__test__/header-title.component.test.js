import React from 'react';
import { HeaderTitle } from '../header-title.component';
import { shallow } from 'enzyme';

describe('<HeaderTitle />', () => {
  describe('Shapshot', () => {
    test('should render <HeaderTitle /> without children', () => {
      const wrapper = shallow(
        <HeaderTitle />
      );

      expect(wrapper).toMatchSnapshot();
    });

    test('should render <HeaderTitle /> with children', () => {
      const wrapper = shallow(
        <HeaderTitle>
          <div>mocks children</div>
        </HeaderTitle>
      );

      expect(wrapper).toMatchSnapshot();
    });
  });
});