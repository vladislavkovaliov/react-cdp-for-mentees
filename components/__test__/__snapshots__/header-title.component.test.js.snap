// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderTitle /> Shapshot should render <HeaderTitle /> with children 1`] = `
<div
  className="header__title"
>
  <p>
    netflixroulette
  </p>
  <div>
    mocks children
  </div>
</div>
`;

exports[`<HeaderTitle /> Shapshot should render <HeaderTitle /> without children 1`] = `
<div
  className="header__title"
>
  <p>
    netflixroulette
  </p>
</div>
`;
